framework/platform/hpp_window.h

```diff
@@ -46,6 +46,11 @@ class HPPWindow : private vkb::Window
 	{
 		return static_cast<vk::SurfaceKHR>(create_surface(static_cast<VkInstance>(instance), static_cast<VkPhysicalDevice>(physical_device)));
 	}
+
+	bool get_display_present_info(vk::DisplayPresentInfoKHR *info, uint32_t src_width, uint32_t src_height) const
+	{
+		return vkb::Window::get_display_present_info(reinterpret_cast<VkDisplayPresentInfoKHR *>(info), src_width, src_height);
```

Comment:
Unfortunately, you should not explicitly call vkb::Window::get_display_present_info, as that would really just call that function.
Please add
`	using vkb::Window::get_display_present_info;`
to the using-section in this file, and remove the explicit `vkb::Window::` here to really get the actual override called.

Reply:
Thanks. That was a subtlety that had passed me by the first time around. Following what's actually going on with the class hierarchies is a lot trickier when they are reinterpret_cast'ed to other things.